BackofficeBundle/Resources/public/less/role.less

```diff
@@ -0,0 +1,27 @@
+.role-table {
+    border-collapse: collapse;
+	display: table;
+}
+.role-row {
```

Comment:
role-row is always in an element with class role-table can you use less syntax to add role-row property in role-table same for other selector